Updates/0332_creature_ai_texts_broadcast_text_id.sql

```diff
@@ -0,0 +1,126 @@
+# ################################################################################
+# TABLE creature_ai_texts
+# ################################################################################
+
+# --------------------------------------------------------------------------------
+# ## tab -104	=> "Who challenges Naias? Puny $r, you are little better than those mindless trolls I have played against each other like so many pieces on a game board!"
+
+# == mal [13560] "Who challenges Naias? Puny $r, you are little better than those mindless trolls I have played against each other like so many pieces on a game board!"
+
+UPDATE creature_ai_texts SET broadcast_text_id = 13560 WHERE entry = -104;
+
+# --------------------------------------------------------------------------------
+# ## tab -321	=> "Iman m t'ief fu Fus'obeah italaf"
```

Comment:
https://imgur.com/gallery/qHLnt

1910	Have a taste of Frostmane magics.	Have a taste of Frostmane magics.	14

Reply:
Nice ! So, it is also possible to find some IDs with the "comment" field in tables : look for the npc quotes on wowhead, and sometimes in the comments from players.

Reply:
Or in creature_text table